tests/periph_spi/main.c

```diff
@@ -91,14 +91,14 @@ int parse_spi_dev(int argc, char **argv)
         return -1;
     }
     if (argc >= 4) {
-        spi_mode = argv[2][0] - '0';
+        spi_mode = argv[3][0] - '0';
```

Comment:
Yes I think so. I tried to change as little as possible from the original code. Does this line hurt you?